build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```
@@ -95,7 +95,8 @@ extraJavaModuleInfo {
     }
     module("com.google.guava:guava", "com.google.common") {
         exportAllPackages()
-        requireAllDefinedDependencies()
+        //requireAllDefinedDependencies()
```

Remove comment

---

mmh, that is the quick fix by @jjohannes to get it working. Maybe we add a text to the comment instead of removing it
```suggestion
        //requireAllDefinedDependencies() <- That is currently needed based on a limitation of the Gradlex plugin
```

---

Suggestion:

```
This module is used on the 'annotation processor path'. requireAllDefinedDependencies() does not work reliably in this context. Hence, we define the 'requires' directly. See: https://github.com/gradlex-org/java-module-dependencies/issues/66 
```